sdk/authentication.ts

```diff
@@ -18,11 +18,16 @@ class Authentication {
   public async getJWS(
     scopes: JWTScopes = [JWTScope.ReadAll, JWTScope.WriteAll],
     expiresIn = '30s',
-    embed?: EmbedParams
+    embed?: EmbedParams,
+    checkoutSessionId?: string
   ): Promise<string> {
     const keyid: string = await this.getKeyId()
     const claims = { scopes }
 
+    if (checkoutSessionId) {
+      claims['checkout_session_id'] = checkoutSessionId
+    }
```

Comment:
I am not sure if there's a more elegant way to do this.

Reply:
Should we check this is a valid GUID? It will error on the API, but might be better feedback at this level.